### ISSUE TITLE
Hospitality: Determining frequency of visits per faction and influencing it for modded faction

### ISSUE THREAD

**Issue by Euphoric** [OrionFive/Hospitality#626]
I'm wondering if it is possible to determine frequency at which specific faction initiates visits. I've tried to read the code, but there isn't anything obvious. Only thing I can tell that factions that have closest settlement have higher chance and that royal titles also play a role.

For example, in my current game, I'm getting visited by The Empire every few days. But a custom faction I've added does not visit me at all, despite having settlement closer than empire. Should I be looking for some specific logs?

I would also like to know if it possible to change frequency of visits for custom-made faction. Maybe even change it if they are neutral/allies.

**Comment by OrionFive** [comment]
The way it works right now is that if you add a faction mid game, you have to either get very lucky, or just invite them with the Comms Console.
If you want to see less of a faction, send them away.

**Comment by Euphoric** [comment]
No. This faction has been there from the start of the game. And I'm seeing them a lot less compared to the empire or some primitive tribes.

As for seeing the faction less. What I'm talking about is if I want to create a mod that adds custom faction I want to make it something like "This faction doesn't like to interact with outsiders, so it won't come for visit often, if at all."

**Comment by OrionFive** [comment]
Yeah, visit frequency is determined by the distance of their closest base.

I could factor Ideology in, like how open they are for other cultures. Although I don't think many players will understand or notice that link.
